packages/syft/src/syft/service/action/action_service.py

```diff
@@ -102,9 +102,12 @@ def _set(
 
         if isinstance(action_object, ActionObject):
             action_object.syft_created_at = DateTime.now()
+            action_object._clear_cache(clear_reprs=True)
```

Comment:
Do we expect to clear repr in all cases ? I think we want to store the repr but truncate it if the repr is too large.

Reply:
hmm yes maybe we can just truncate the repr indeed

Reply:
Yes, lets set a max repr to like 100 characters or something.